hasher-matcher-actioner/Makefile

```diff
@@ -8,7 +8,7 @@ shell-or-die = $\
 
 USER ?= $(call shell-or-die,whoami)
 REPOSITORY_NAME = hma-lambda-dev
-DOCKER_TAG ?= ${USER} # todo - change to ${USER}-latest?
+DOCKER_TAG ?= ${USER}# todo - change to ${USER}-latest?
```

Comment:
the fact that could matter is a "someone new to Makefile"'s worst nightmare haha